src/views/TechnicalAssistance/TrbAdminTeamHome.tsx

```diff
@@ -390,7 +390,22 @@ function TrbExistingRequestsTable({ requests }: TrbRequestsTableProps) {
             >
               {t('adminTeamHome.actions.addDate')}
             </UswdsReactLink>
-          )
+          ),
+        sortType: (a, b) => {
```

Comment:
is this sort order something the BE could just return by default?

Reply:
It's probably worth treating pagination, filtering, and ordering as a single effort on the backend. Currently, in cases like this, we basically return a bunch of data to the FE and let the FE handle sorting, searching (live-typing filtering results), and pagination.

So, I guess I'd say "probably not necessary here, but we could re-visit how we return large chunks/arrays of data to see if we could make improvements in the future" 😁

Reply:
Sounds great! Thanks Clay

Reply:
> is this sort order something the BE could just return by default?

@samoddball I _really_ appreciate this type of comment and consideration from the backend. Overall I agree with what you're getting at. We just happen to do stuff like this in FE atm. Looking forward.